Sanity/postgresql-odbc/main.fmf

```diff
@@ -0,0 +1,23 @@
+summary: Basic smoke-test for accessing PostgreSQL via unixODBC
+contact:
+- Honza Horak <hhorak@redhat.com>
+component:
+- python3-pyodbc
+- unixODBC
+- postgresql-odbc
+- postgresql
+test: ./runtest.sh
+framework: beakerlib
+recommend:
+- python3-pyodbc
```

Comment:
The same duplication.

Reply:
What duplication do you mean?

Reply:
I think this is expected to have it listed twice in two different sections.

Reply:
It could be a good idea for improvement, but for now, it's necessary, I am right?

Reply:
agree